bundles/admin/admin-announcements/resources/locale/sv.js

```diff
@@ -33,6 +33,15 @@ Oskari.registerLocalization(
                 "deleteAnnouncementConfirm": "Radera avisering?",
                 "cancelAnnouncementConfirm": "Är du säker?"
             },
+            "tool": {
+                "buttonLabel": "Välj Aviseringar",
+                "popup": {
+                    "title": "Aviseringar",
+                    "close": "Stäng"
+                },
+                "announcementsName": "Namn",
+                "announcementsTime": "VaDatumintervalllid"
```

Comment:
VaDatumintervalllid looks weird with CamelCasing. Can you confirm this is correct?